### PR TITLE
Add stale action for open issues

### PR DESCRIPTION
**Pull request by marcosmarxm**
This PR adds 2 stale actions:
* issues without frozen label not updated in 180+ days and close after 20 days ([currently 280 issues](https://github.com/airbytehq/airbyte/issues?q=is%3Aissue+is%3Aopen+label%3Acommunity+updated%3A%3C2023-09-19+-label%3Afrozen))
* issues with frozen label not updated in 365+ days and close after 20 days([currently 0 issues](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Afrozen+updated%3A%3C2023-03-25))

### CODE REVIEW

File: .github/workflows/stale-community-issues.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Stale issue action
+on:
+  schedule:
+    - cron: "30 1 * * *"
```
Comment (bleonard) [inline .github/workflows/stale-community-issues.yaml:4]: I'd run these ate different times. Maybe this one at 15 and the other at 45.

File: .github/workflows/stale-community-issues.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          any-of-labels: "community"
+          exempt-issue-labels: "frozen"
```
Comment (bleonard) [inline .github/workflows/stale-community-issues.yaml:16]: I feel like maybe there are some more exempt ones here and below. The ones that comes to mind the ones that are still on the roadmap. https://github.com/orgs/airbytehq/projects/37/views/1
We could run scripts to tag those automatically.

Comment (bleonard) [reply .github/workflows/stale-community-issues.yaml:16]: I'm not sure if there are multiple tags here like `roadmap` and `somethingelse` or there should just be one like `evergreen` or something.

Comment (marcosmarxm) [reply .github/workflows/stale-community-issues.yaml:16]: Roadmap issues are usually created by the Airbyte team and won't have the `community` label.

File: .github/workflows/stale-routed-issues.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          any-of-labels: "frozen"
```
Comment (marcosmarxm) [inline .github/workflows/stale-routed-issues.yaml:15]: In this case it only apply stale action to issues were triaged by the engineering teams.
Roadmap and other issues I don't think will be enter this workflow.

File: .github/workflows/stale-community-issues.yaml (hunk 2 above)
Comment (bleonard) [reply .github/workflows/stale-community-issues.yaml:16]: why is `frozen` exempt though. Just so it doesn't compete with the other timeline? What about ones from the community that are ebing worked on. Maybe those get assigned and stuff so they don't get stale?

**Review by bleonard — APPROVED** [review]
Overall, whatever you think. Go for it.

**Pull request merged**